fulp_modules/features/toys/code/plushies.dm

```diff
@@ -87,3 +87,25 @@
 	icon = 'fulp_modules/features/toys/icons/toys.dmi'
 	icon_state = "beefman"
 	squeak_override = list('sound/effects/meatslap.ogg'=1)
+
+//Do your cooldown changes here.
+#define BEEFPLUSHIE_COOLDOWN_TIME (2 MINUTES)
+
+/obj/item/toy/plush/beefplushie/living
+	desc = "It looks oddly alive. You feel like you should pet it."
+	COOLDOWN_DECLARE(beefplushie_cooldown)
+
+//When used in hand.
+/obj/item/toy/plush/beefplushie/living/attack_self(mob/user)
+	. = ..()
+	if(!COOLDOWN_FINISHED(src, beefplushie_cooldown))
+		user.balloon_alert(user, "not ready yet!")
+		return
+	user.balloon_alert(user, "producing meat")
```

Comment:
why are you making the user balloon alert themselves

Reply:
this isn't resolved why did you mark it such

Reply:
thought i fixed it, how should i do it?

Reply:
okay cool, i think i was just blind, and forgot to fix one of them i'll fix it when i get home